### PR TITLE
feat(assessments): brand polish + branded results report (Phase 1)

### PR DESCRIPTION
**Pull request by MonksKoala**
Closes Jeff's 2026-06-05 asks: **(1)** polish the look & feel of the assessments, **(2)** "I see raw data — I want the PDF that gets sent to them."

## What
- **Participant quiz polish** — branded app shell (logo + Four-Decisions progress), polished section title/intro slides, restyled Likert slider (kept per D4, not buttons).
- **Branded results report ("the PDF")** — Esperto-anatomy web report (cover → overall → section breakdown → scores → recommendations → conclusion), **adaptive per template** (Rockefeller checklist + tier band / QSP+LVA neutral "Submitted" / SU Full domains + ScaleUp + recs), **Print → PDF**. Coach/admin-only view; **replaces** the raw `stableKey` view.

## How / safety
- **Zero schema migrations.** Reuses the frozen `ScoreResult` + wider `select` clauses.
- **Phased rollout (H13):** Phase 1 (this PR) is additive — the raw inline view + `/result` API stay (deprecated + hit-logged); Phase 2 (separate release) removes them after telemetry shows `/result` is unused.
- Report is **invited-only**, **coach/admin-gated** (`getApiActor` + `canManageCampaign`, authorized in one transaction), **audited**, **provenance-stamped**, **`no-store` + rate-limited**.
- Brand scoped per **ADR-0005** (`.su-public-brand` report / `.su-assessment-brand` quiz; no global token leak).

## Process
Designed via brainstorm + frontend-design (mockups approved), grilled (`/grill-with-docs`, G1–G8), adversarially reviewed by **claudex** (3 rounds → hardening **H1–H17**), then Greptile.

Spec: `docs/specs/v7.6/13` · Plan: `13a` · Decision: `ADR-0007`. *(Code commits follow on this branch.)*

🤖 Generated with [Claude Code](https://claude.com/claude-code)

<h3>Greptile Summary</h3>

Adds brand polish to the participant quiz (app-shell header with logo + Four-Decisions segmented progress, polished section intro slides) and introduces the coach/admin-gated branded results report (`BrandedReport`) as a printable, PII-protected per-respondent view that replaces the raw `stableKey` inline view.

- **Branded results report** — new `(report)` route group with its own layout (no portal chrome), server-component page with actor check + `canManageCampaign` in a single transaction, `Cache-Control: no-store, private` via middleware, audit log on every successful view, and adaptive rendering per template.
- **Quiz shell polish** — `AssessmentShellHeader` replaces the plain section counter across both the public-quiz and org-survey flows; `SectionPager` gains a styled intro slide; both callers now pass `assessmentName` and `companyName`.
- **Phase 1 additive rollout** — the legacy raw `/result` API and inline \"Raw data\" button are kept (de-emphasized, hit-logged) and gated for Phase 2 removal after telemetry shows zero usage.

<h3>Confidence Score: 4/5</h3>

Safe to merge after fixing the missing ESLint suppression comment in AssessmentShellHeader.tsx, which would fail the project CI build step.

The auth, caching, and data-loading logic is well-constructed and hardened across multiple review rounds. The one concrete defect is a missing eslint-disable-next-line comment in AssessmentShellHeader.tsx for its bare img element — the same suppression BrandedReport.tsx uses for both of its img tags. CLAUDE.md mandates fixing all ESLint warnings/errors before pushing and Vercel treats them as build failures.

src/src/components/assessments/AssessmentShellHeader.tsx — the bare img at line 44 needs the eslint-disable-next-line @next/next/no-img-element comment and width/height attributes, matching the pattern used throughout BrandedReport.tsx.

<h3>Important Files Changed</h3>

| Filename | Overview |
|----------|----------|
| src/src/components/assessments/AssessmentShellHeader.tsx | New branded quiz app-shell header — uses bare img without eslint-disable comment, triggering the no-img-element ESLint rule enforced elsewhere in the codebase |
| src/src/app/(report)/assessments/[id]/respondents/[respondentId]/report/page.tsx | New coach/admin-gated per-respondent report page — double-protected, auth+fetch in one transaction, PII headers via middleware, rate-limit uses permissive standard tier |
| src/src/lib/assessments/respondent-report.ts | New authorized data loader — canManageCampaign + fetch in a single transaction, degrades gracefully on malformed result |
| src/src/components/assessments/BrandedReport.tsx | New adaptive branded report component — template-aware, graceful degradation throughout, section chip correctly gated on showCheckmarks |
| src/src/lib/assessments/report-presentation.ts | New pure presentation helpers — all total-tolerant of malformed input; SU Full correctly prioritizes scaleUpScore over overallAvg |
| src/src/middleware.ts | Adds Cache-Control: no-store, private for the report page URL; regex correctly scoped and only applies to authenticated responses |
| src/src/components/assessments/CampaignDetail.tsx | Adds primary View report link as plain anchor (avoids Next.js PII prefetch) and de-emphasizes legacy Raw data button for Phase 1 |
| src/src/components/assessments/section-pager.tsx | Replaces plain section counter with branded AssessmentShellHeader + polished intro slide; assessmentName/companyName wired from both callers |
| src/src/app/(report)/layout.tsx | Minimal brand-scoped layout for the report route group — no portal nav/sidebar, imports scoped CSS |

</details>

<h3>Sequence Diagram</h3>

```mermaid
sequenceDiagram
    participant Coach as Coach / Admin
    participant MW as Middleware (withAuth)
    participant Page as RespondentReportPage
    participant Loader as getRespondentReport
    participant DB as Prisma ($transaction)
    participant Audit as logAudit

    Coach->>MW: GET /assessments/[id]/respondents/[rid]/report
    MW->>MW: authorized() check token
    MW->>MW: Set Cache-Control: no-store, private
    MW->>Page: forward request
    Page->>Page: getApiActor() actor check
    Page->>Page: checkRateLimitAsync(report:ip)
    Page->>Loader: getRespondentReport(db, actor, id, respondentId)
    Loader->>DB: $transaction canManageCampaign + findFirst
    DB-->>Loader: submission or null/forbidden
    Loader-->>Page: status ok/forbidden/not-found
    alt forbidden or not-found
        Page-->>Coach: notFound() enumeration-safe 404
    else ok
        Page->>Audit: logAudit VIEW_REPORT submissionId
        Page-->>Coach: BrandedReport + PrintReportButton
    end
```

<details><summary><h3>Comments Outside Diff (5)</h3></summary>

1. `docs/specs/v7.6/13a-assessment-brand-and-results-report-implementation-plan.md`, line 305-307 ([link](https://github.com/jcbdelo26/scaling-up-platform-v2/blob/639306c4b87c575cf64d5bc2806a1e2cc74a606d/docs/specs/v7.6/13a-assessment-brand-and-results-report-implementation-plan.md#L305-L307)) 

   <a href="#">P1</a> **H17 audit-action verification has no test gate**

   Task 4 Step 3 defers the `AuditLog.action` column-type check to implementation time with the note "Verified before implementation," but Task 4 Step 1's failing tests only cover auth/cache/rate-limit scenarios — there is no test that enforces the zero-migration constraint on the audit action. If `AuditLog.action` is a Prisma enum (not a plain `String`), adding `VIEW_REPORT` silently becomes a schema migration, directly violating D5/H17. The safe default (`EXPORT`) is already specified, but the "use VIEW_REPORT only if it's a String column" branch has no automated guard. Consider either (a) making the Step 1 test explicitly assert that `EXPORT` is used, or (b) adding a pre-implementation check step that inspects the Prisma schema for `AuditLog.action`'s type and documents the result before any code is written for Task 4.

   <a href="https://app.greptile.com/ide/claude-code?prompt=This%20is%20a%20comment%20left%20during%20a%20code%20review.%0APath%3A%20docs%2Fspecs%2Fv7.6%2F13a-assessment-brand-and-results-report-implementation-plan.md%0ALine%3A%20305-307%0A%0AComment%3A%0A**H17%20audit-action%20verification%20has%20no%20test%20gate**%0A%0ATask%204%20Step%203%20defers%20the%20%60AuditLog.action%60%20column-type%20check%20to%20implementation%20time%20with%20the%20note%20%22Verified%20before%20implementation%2C%22%20but%20Task%204%20Step%201's%20failing%20tests%20only%20cover%20auth%2Fcache%2Frate-limit%20scenarios%20%E2%80%94%20there%20is%20no%20test%20that%20enforces%20the%20zero-migration%20constraint%20on%20the%20audit%20action.%20If%20%60AuditLog.action%60%20is%20a%20Prisma%20enum%20%28not%20a%20plain%20%60String%60%29%2C%20adding%20%60VIEW_REPORT%60%20silently%20becomes%20a%20schema%20migration%2C%20directly%20violating%20D5%2FH17.%20The%20safe%20default%20%28%60EXPORT%60%29%20is%20already%20specified%2C%20but%20the%20%22use%20VIEW_REPORT%20only%20if%20it's%20a%20String%20column%22%20branch%20has%20no%20automated%20guard.%20Consider%20either%20%28a%29%20making%20the%20Step%201%20test%20explicitly%20assert%20that%20%60EXPORT%60%20is%20used%2C%20or%20%28b%29%20adding%20a%20pre-implementation%20check%20step%20that%20inspects%20the%20Prisma%20schema%20for%20%60AuditLog.action%60's%20type%20and%20documents%20the%20result%20before%20any%20code%20is%20written%20for%20Task%204.%0A%0AHow%20can%20I%20resolve%20this%3F%20If%20you%20propose%20a%20fix%2C%20please%20make%20it%20concise.&repo=jcbdelo26%2Fscaling-up-platform-v2&pr=41&platform=github"><picture><source media="(prefers-color-scheme: dark)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaudeDark.svg?v=3"><source media="(prefers-color-scheme: light)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaude.svg?v=3">Fix in Claude Code</picture></a>

2. `docs/specs/v7.6/13-assessment-brand-and-results-report.md`, line 156-158 ([link](https://github.com/jcbdelo26/scaling-up-platform-v2/blob/639306c4b87c575cf64d5bc2806a1e2cc74a606d/docs/specs/v7.6/13-assessment-brand-and-results-report.md#L156-L158)) 

   <a href="#">P2</a> **MULTI_CHOICE render format unspecified for Additional responses (H9)**

   The Additional responses section joins `rawAnswers` to `questionsByKey` for `TEXT/NUMBER/MULTI_CHOICE` types, but the spec doesn't define how a `MULTI_CHOICE` answer (stored as `string[]` in the `Json` column) should be rendered. Without an explicit format, an implementer may output raw JSON or call `.toString()`. Specify the intended display — e.g., a comma-joined list or a bulleted list of selected options — to avoid an unpolished default surfacing in the branded report.

   Note: If this suggestion doesn't match your team's coding style, reply to this and let me know. I'll remember it for next time!

   <a href="https://app.greptile.com/ide/claude-code?prompt=This%20is%20a%20comment%20left%20during%20a%20code%20review.%0APath%3A%20docs%2Fspecs%2Fv7.6%2F13-assessment-brand-and-results-report.md%0ALine%3A%20156-158%0A%0AComment%3A%0A**MULTI_CHOICE%20render%20format%20unspecified%20for%20Additional%20responses%20%28H9%29**%0A%0AThe%20Additional%20responses%20section%20joins%20%60rawAnswers%60%20to%20%60questionsByKey%60%20for%20%60TEXT%2FNUMBER%2FMULTI_CHOICE%60%20types%2C%20but%20the%20spec%20doesn't%20define%20how%20a%20%60MULTI_CHOICE%60%20answer%20%28stored%20as%20%60string%5B%5D%60%20in%20the%20%60Json%60%20column%29%20should%20be%20rendered.%20Without%20an%20explicit%20format%2C%20an%20implementer%20may%20output%20raw%20JSON%20or%20call%20%60.toString%28%29%60.%20Specify%20the%20intended%20display%20%E2%80%94%20e.g.%2C%20a%20comma-joined%20list%20or%20a%20bulleted%20list%20of%20selected%20options%20%E2%80%94%20to%20avoid%20an%20unpolished%20default%20surfacing%20in%20the%20branded%20report.%0A%0AHow%20can%20I%20resolve%20this%3F%20If%20you%20propose%20a%20fix%2C%20please%20make%20it%20concise.&repo=jcbdelo26%2Fscaling-up-platform-v2&pr=41&platform=github"><picture><source media="(prefers-color-scheme: dark)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaudeDark.svg?v=3"><source media="(prefers-color-scheme: light)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaude.svg?v=3">Fix in Claude Code</picture></a>

3. `docs/specs/v7.6/13a-assessment-brand-and-results-report-implementation-plan.md`, line 222 ([link](https://github.com/jcbdelo26/scaling-up-platform-v2/blob/639306c4b87c575cf64d5bc2806a1e2cc74a606d/docs/specs/v7.6/13a-assessment-brand-and-results-report-implementation-plan.md#L222)) 

   <a href="#">P2</a> **Hardcoded local developer path**

   The line `Run commands from /Users/diushianstand/Scaling-up-platform-v2/src` embeds a specific developer's machine path. Any contributor (or agentic worker) following this plan on a different machine will hit a confusing dead reference. `CLAUDE.md` already records the canonical source path and the build commands — suggest replacing this sentence with a reference to `CLAUDE.md`'s "Development Commands" section.

   Note: If this suggestion doesn't match your team's coding style, reply to this and let me know. I'll remember it for next time!

   <a href="https://app.greptile.com/ide/claude-code?prompt=This%20is%20a%20comment%20left%20during%20a%20code%20review.%0APath%3A%20docs%2Fspecs%2Fv7.6%2F13a-assessment-brand-and-results-report-implementation-plan.md%0ALine%3A%20222%0A%0AComment%3A%0A**Hardcoded%20local%20developer%20path**%0A%0AThe%20line%20%60Run%20commands%20from%20%2FUsers%2Fdiushianstand%2FScaling-up-platform-v2%2Fsrc%60%20embeds%20a%20specific%20developer's%20machine%20path.%20Any%20contributor%20%28or%20agentic%20worker%29%20following%20this%20plan%20on%20a%20different%20machine%20will%20hit%20a%20confusing%20dead%20reference.%20%60CLAUDE.md%60%20already%20records%20the%20canonical%20source%20path%20and%20the%20build%20commands%20%E2%80%94%20suggest%20replacing%20this%20sentence%20with%20a%20reference%20to%20%60CLAUDE.md%60's%20%22Development%20Commands%22%20section.%0A%0AHow%20can%20I%20resolve%20this%3F%20If%20you%20propose%20a%20fix%2C%20please%20make%20it%20concise.&repo=jcbdelo26%2Fscaling-up-platform-v2&pr=41&platform=github"><picture><source media="(prefers-color-scheme: dark)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaudeDark.svg?v=3"><source media="(prefers-color-scheme: light)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaude.svg?v=3">Fix in Claude Code</picture></a>

4. `docs/specs/v7.6/13-assessment-brand-and-results-report.md`, line 174 ([link](https://github.com/jcbdelo26/scaling-up-platform-v2/blob/639306c4b87c575cf64d5bc2806a1e2cc74a606d/docs/specs/v7.6/13-assessment-brand-and-results-report.md#L174)) 

   <a href="#">P2</a> **Phase 2 gate lacks a concrete threshold**

   H13 says Phase 2 ships "only after telemetry shows `/result` is no longer hit," but the spec doesn't define what that means in practice (e.g., zero hits for 30 consecutive days, or below N/day for a week). Without a measurable exit criterion the telemetry counter works but Phase 2 may be deferred indefinitely. Recommend pinning a concrete definition — even a provisional one like "zero `assessment.report.old_result_api.hit` events in any rolling 14-day window" — so the Phase 2 release decision is unambiguous.

   <a href="https://app.greptile.com/ide/claude-code?prompt=This%20is%20a%20comment%20left%20during%20a%20code%20review.%0APath%3A%20docs%2Fspecs%2Fv7.6%2F13-assessment-brand-and-results-report.md%0ALine%3A%20174%0A%0AComment%3A%0A**Phase%202%20gate%20lacks%20a%20concrete%20threshold**%0A%0AH13%20says%20Phase%202%20ships%20%22only%20after%20telemetry%20shows%20%60%2Fresult%60%20is%20no%20longer%20hit%2C%22%20but%20the%20spec%20doesn't%20define%20what%20that%20means%20in%20practice%20%28e.g.%2C%20zero%20hits%20for%2030%20consecutive%20days%2C%20or%20below%20N%2Fday%20for%20a%20week%29.%20Without%20a%20measurable%20exit%20criterion%20the%20telemetry%20counter%20works%20but%20Phase%202%20may%20be%20deferred%20indefinitely.%20Recommend%20pinning%20a%20concrete%20definition%20%E2%80%94%20even%20a%20provisional%20one%20like%20%22zero%20%60assessment.report.old_result_api.hit%60%20events%20in%20any%20rolling%2014-day%20window%22%20%E2%80%94%20so%20the%20Phase%202%20release%20decision%20is%20unambiguous.%0A%0AHow%20can%20I%20resolve%20this%3F%20If%20you%20propose%20a%20fix%2C%20please%20make%20it%20concise.&repo=jcbdelo26%2Fscaling-up-platform-v2&pr=41&platform=github"><picture><source media="(prefers-color-scheme: dark)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaudeDark.svg?v=3"><source media="(prefers-color-scheme: light)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaude.svg?v=3">Fix in Claude Code</picture></a>

5. `docs/specs/v7.6/13-assessment-brand-and-results-report.md`, line 176 ([link](https://github.com/jcbdelo26/scaling-up-platform-v2/blob/639306c4b87c575cf64d5bc2806a1e2cc74a606d/docs/specs/v7.6/13-assessment-brand-and-results-report.md#L176)) 

   <a href="#">P2</a> **Report-route rate limit rate is unspecified**

   H15 mandates `withRateLimit` on the report route but doesn't pin a specific rate (requests / window / scope). Other endpoints are explicit (e.g. survey submission is 20 req/min per IP). The report serves PII and leaving the rate implicit means the implementer falls back to whatever default `withRateLimit` applies, which may be too permissive. Recommend specifying a concrete limit — e.g., 30 req/min per authenticated user — so the rate is a reviewed decision rather than an implementation default.

   <a href="https://app.greptile.com/ide/claude-code?prompt=This%20is%20a%20comment%20left%20during%20a%20code%20review.%0APath%3A%20docs%2Fspecs%2Fv7.6%2F13-assessment-brand-and-results-report.md%0ALine%3A%20176%0A%0AComment%3A%0A**Report-route%20rate%20limit%20rate%20is%20unspecified**%0A%0AH15%20mandates%20%60withRateLimit%60%20on%20the%20report%20route%20but%20doesn't%20pin%20a%20specific%20rate%20%28requests%20%2F%20window%20%2F%20scope%29.%20Other%20endpoints%20are%20explicit%20%28e.g.%20survey%20submission%20is%2020%20req%2Fmin%20per%20IP%29.%20The%20report%20serves%20PII%20and%20leaving%20the%20rate%20implicit%20means%20the%20implementer%20falls%20back%20to%20whatever%20default%20%60withRateLimit%60%20applies%2C%20which%20may%20be%20too%20permissive.%20Recommend%20specifying%20a%20concrete%20limit%20%E2%80%94%20e.g.%2C%2030%20req%2Fmin%20per%20authenticated%20user%20%E2%80%94%20so%20the%20rate%20is%20a%20reviewed%20decision%20rather%20than%20an%20implementation%20default.%0A%0AHow%20can%20I%20resolve%20this%3F%20If%20you%20propose%20a%20fix%2C%20please%20make%20it%20concise.&repo=jcbdelo26%2Fscaling-up-platform-v2&pr=41&platform=github"><picture><source media="(prefers-color-scheme: dark)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaudeDark.svg?v=3"><source media="(prefers-color-scheme: light)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixInClaude.svg?v=3">Fix in Claude Code</picture></a>
</details>

<a href="https://app.greptile.com/ide/claude-code?prompt=Fix%20the%20following%202%20code%20review%20issues.%20Work%20through%20them%20one%20at%20a%20time%2C%20proposing%20concise%20fixes.%0A%0A---%0A%0A%23%23%23%20Issue%201%20of%202%0Asrc%2Fsrc%2Fcomponents%2Fassessments%2FAssessmentShellHeader.tsx%3A44%0A**Missing%20%60eslint-disable%60%20for%20%60%3Cimg%3E%60%20element**%0A%0A%60BrandedReport.tsx%60%20uses%20%60%7B%2F*%20eslint-disable-next-line%20%40next%2Fnext%2Fno-img-element%20*%2F%7D%60%20before%20every%20%60%3Cimg%3E%60%20tag%20%28lines%20285%20and%20580%29%20because%20the%20%60%40next%2Fnext%2Fno-img-element%60%20rule%20is%20active%20in%20this%20project.%20This%20file%20uses%20%60%3Cimg%3E%60%20without%20that%20suppression%20comment%20and%20also%20omits%20the%20required%20%60width%60%2F%60height%60%20attributes.%20Per%20CLAUDE.md%3A%20%22Fix%20ALL%20warnings%20AND%20errors.%20Vercel%20may%20treat%20warnings%20as%20build%20failures.%22%20%E2%80%94%20the%20missing%20comment%20is%20likely%20to%20fail%20the%20ESLint%20step%20of%20the%20Vercel%20build.%0A%0A%23%23%23%20Issue%202%20of%202%0Asrc%2Fsrc%2Fapp%2F%28report%29%2Fassessments%2F%5Bid%5D%2Frespondents%2F%5BrespondentId%5D%2Freport%2Fpage.tsx%3A69-72%0A**Report%20rate%20limit%20uses%20the%20most%20permissive%20tier%20%28100%20req%2Fmin%20per%20IP%29**%0A%0A%60RateLimits.standard%60%20is%20defined%20as%20100%20requests%20per%20minute%20%E2%80%94%20the%20same%20budget%20used%20for%20general%20API%20listing%20endpoints.%20For%20a%20page%20that%20renders%20a%20respondent's%20full%20PII%20%28scores%2C%20answers%2C%20job%20title%29%2C%20a%20tighter%20limit%20would%20be%20more%20appropriate%3A%20%60RateLimits.registration%60%20%2820%2Fmin%29%20or%20%60RateLimits.auth%60%20%2810%2Fmin%29%20both%20correspond%20to%20sensitive%20endpoints%20in%20this%20codebase.%20An%20authenticated%20coach%20viewing%20their%20own%20campaign's%20reports%20would%20rarely%20exceed%2020%20requests%20per%20minute%20organically.%0A%0A&repo=jcbdelo26%2Fscaling-up-platform-v2&pr=41&platform=github"><picture><source media="(prefers-color-scheme: dark)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixAllInClaudeDark.svg?v=3"><source media="(prefers-color-scheme: light)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/FixAllInClaude.svg?v=3">Fix All in Claude Code</picture></a>

<sub>Reviews (8): Last reviewed commit: ["docs: SoT bump for assessment brand + br..."](https://github.com/jcbdelo26/scaling-up-platform-v2/commit/beab12497c46dc20369556f41a2f6b9544b09c4e) | [Re-trigger Greptile](https://app.greptile.com/api/retrigger?id=35908941)</sub>

### CODE REVIEW

**Comment by MonksKoala** [comment]
@greptile review

**Comment by MonksKoala** [comment]
@greptile review

File: src/src/components/assessments/BrandedReport.tsx [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+                      style={{ display: "none" }}
+                    />
+                  )}
+                  <h3 className="su-report-card-title">
+                    {parsed?.name ?? ps.name ?? ps.stableKey}
+                  </h3>
+                  <span className="su-report-card-chip">
+                    {ps.achievedCount} / {ps.totalCount}
+                  </span>
+                </div>
```
[the remark this answers is not in the document]
Comment (MonksKoala) [reply src/src/components/assessments/BrandedReport.tsx]: Fixed — the section chip is now gated on `showCheckmarks` (passThreshold>0), so neutral templates (QSP/LVA) no longer show a meaningless N/N. Test added. Thanks!
